### PR TITLE
Enrich pod sandbox

### PR DESCRIPTION
**Pull request by NDStrahilevitz**
containers: add sandbox container awareness

Add a new argument "pod_sandbox" and event field to container events to
indicate the container's type in its pod.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -104,3 +104,5 @@ require (
 )
 
 replace github.com/kubernetes/cri-api => k8s.io/cri-api v0.23.5-rc.0
+
+replace github.com/aquasecurity/tracee/types => github.com/NDStrahilevitz/tracee/types v0.0.0-20230116133302-d841182726fe
```
Comment (NDStrahilevitz) [inline go.mod:108]: Remove and replace with a bump later.

**Review by rafaeldtinoco — APPROVED** [review]
LGTM. @geyslan could you double-check if @NDStrahilevitz can merge this tomorrow? I don't think there will be an impact on the multi-scopes PR, but I'm double-checking all PRs with you like @yanivagman requested (and I think its good this way also).

File: go.mod (hunk 1 above)
Comment (rafaeldtinoco) [reply go.mod:108]: Yes, please remove this and make sure the git log is properly breaking lines and aligned (not sure it is, the web interface is weird). Tks

**Comment by geyslan** [comment]
> LGTM. @geyslan could you double-check if @NDStrahilevitz can merge this tomorrow? I don't think there will be an impact on the multi-scopes PR, but I'm double-checking all PRs with you like @yanivagman requested (and I think its good this way also).

Just a minor conflict, **go for it**.



```diff
diff --cc cmd/tracee-rules/output_test.go
index 25f3ef2e,51b31414..00000000
--- a/cmd/tracee-rules/output_test.go
+++ b/cmd/tracee-rules/output_test.go
@@@ -268,7 -268,7 +268,11 @@@ func TestOutputTemplates(t *testing.T) 
                                        "a":123,"b":"c","d":true,"f":{"123":"456","foo":"bar"}
                                },
                                "Context":{
++<<<<<<< HEAD
 +                                      "timestamp":1321321,"processorId":0,"processId":21312,"threadId":0,"threadStartTime":0,"parentProcessId":0,"hostProcessId":0,"hostThreadId":0,"hostParentProcessId":0,"userId":0,"mountNamespace":0,"pidNamespace":0,"processName":"","hostName":"","cgroupId":0,"containerId":"abbc123","containerImage":"", "containerName":"","podName":"","podNamespace":"","podUID":"","podSandbox":false,"eventId":"0","eventName":"execve","argsNum":0,"returnValue":0,"stackAddresses":null,"args":null,"contextFlags":{"containerStarted":true}
++=======
+                                       "timestamp":1321321,"processorId":0,"processId":21312,"threadId":0,"threadStartTime":0,"parentProcessId":0,"hostProcessId":0,"hostThreadId":0,"matchedScopes":0,"hostParentProcessId":0,"userId":0,"mountNamespace":0,"pidNamespace":0,"processName":"","hostName":"","cgroupId":0,"containerId":"abbc123","containerImage":"", "containerName":"","podName":"","podNamespace":"","podUID":"","eventId":"0","eventName":"execve","argsNum":0,"returnValue":0,"stackAddresses":null,"args":null,"contextFlags":{"containerStarted":true}
++>>>>>>> f29e5bea (feat: multi scopes (set) of filters)
                                },
                                "SigMetadata":{
                                        "ID":"TRC-1","EventName": "stdio","Version":"0.1.0","Name":"Standard Input/Output Over Socket","Description":"Redirection of process's standard input/output to socket","Tags":["linux","container"],"Properties":{"MITRE ATT\u0026CK":"Persistence: Server Software Component","Severity":3}
```

**Pull request merged**